src/rust/inetstack/protocols/layer4/tcp/established/ctrlblk.rs

```diff
@@ -670,6 +670,8 @@ impl SharedControlBlock {
                             seg_len -= 1;
                             self.store_out_of_order_fin(seg_end);
                             seg_end = seg_end - SeqNumber::from(1);
+                            // Clear header FIN and process later.
```

Comment:
What does "process later" mean here? Is this already implicitly happening in the code, or something that will happen from now on?

Reply:
Ah, I can be more clear but it means process it later when the rest of the packets before this sequence number arrive.

Reply:
I see. If it's implicit and easy to understand from reading the code, then we should let the reader rely on the semantics based on code itself instead of explaining/mentioning it here. Else it will leave the reader guessing.

OR - we should explain it fully in this comment.